pages/created_account.js

```diff
@@ -0,0 +1,8 @@
+const { I } = inject();
+
+module.exports = {
+
+  verifyCreatedAccountPageName() {
```

Comment:
Move this method to register page and remove file

Reply:
+